lib/ops/opsservice/update.go

```diff
@@ -226,20 +215,49 @@ func (o *Operator) RotatePlanetConfig(req ops.RotateConfigPackageRequest) (*ops.
 		return nil, trace.NotFound("couldn't find master server: %v", req)
 	}
 
-	ctx.update = updateContext{
-		masterIP:  master.AdvertiseIP,
-		installOp: *installOperation,
-		app:       *updateApp,
-		gravityPath: filepath.Join(
-			state.GravityUpdateDir(node.StateDir()), constants.GravityBin),
-	}
+	dockerConfig := cluster.dockerConfig()
+	checks.OverrideDockerConfig(&dockerConfig,
+		checks.DockerConfigFromSchema(req.Manifest.SystemOptions.DockerConfig()))
 
-	resp, err := cluster.configurePlanetOnNode(
-		ctx, runner, node, etcd, updateApp.Manifest)
+	configVersion := fmt.Sprintf("%v+%v", req.Package.Version, time.Now().UTC().Unix())
```

Comment:
The planet package configuration versioning scheme is `planet-config-<node-name>:<planet-version>`. In order to support rolling updates, the `planet-version` part is updated with a timestamp as metadata and they're compared with a metadata-aware comparator.